boot/zephyr/flash_map.c

```diff
@@ -34,28 +36,22 @@ extern struct device *boot_flash_device;
 static const struct flash_area part_map[] = {
 	{
 		.fa_id = FLASH_AREA_IMAGE_0,
-		.fa_off = 0x20000,
-		.fa_size = 0x20000,
+		.fa_off = FLASH_AREA_IMAGE_0_OFFSET,
```

Comment:
I think this is good for now. It is likely that we will bring in the partition table code from mynewt into mcuboot, which will make all of this quite different, though.